content/get-started/climate-change-and-climate-data-in-california.md

```diff
@@ -36,12 +36,26 @@ These trends will vary in magnitude and impact across California, so it’s impo
-
-- [Adaptation Planning Guide 2.0](https://resilientca.org/apg/)
-- [State of California Sea Level Rise Guidance, 2018 Update](https://www.opc.ca.gov/webmaster/ftp/pdf/agenda_items/20180314/Item3_Exhibit-A_OPC_SLR_Guidance-rd3.pdf)
-- [Planning and Investing for a Resilient California: A Guidebook for State Agencies](http://opr.ca.gov/docs/20180313-Building_a_Resilient_CA.pdf)
+Cal-Adapt data can be used in combination with other data sources to inform a climate vulnerability assessment, which combines projections data, historical data, and other qualitative and quantitative sources to identify how specific climate-related risks will impact a community. Vulnerability assessments are commonly used to identify the populations, infrastructure, and natural systems most at risk of climate change impacts and evaluate the severity of their vulnerability and their ability to adapt. Climate change vulnerability analysis and adaptation planning methods can vary by sector. Regardless of the context, using climate change data to understand vulnerability is an important cornerstone of identifying the best adaptations to undertake to address the projected risks.
 
-You can find a list of additional climate change adaptation planning resources on the [California Adaptation Clearinghouse](https://resilientca.org/apg/adaptation-planning-resources/).
+The 2020 Adaptation Planning Guide (APG) provides guidance to local governments on local adaptation and resiliency planning in California, including how to use Cal-Adapt to inform a vulnerability assessment. The APG breaks down adaptation planning into four phases:
 
-If you want recommendations for analyzing and interpreting climate projections, check out [this Cal-Adapt guidance on best practices](help/get-started/best-practices).
+1. Explore, Define, and Initiate
+2. Assess Vulnerability
+3. Define Adaptation Framework and Strategies
+4. Implement, Monitor, Evaluate, and Adjust
+ 
+While following this process, it is most useful to start examining climate change data from Cal-Adapt in a preliminary analysis during Phase 1 to understand the scope of the assessment you are about to undertake, and then to conduct a more thorough technical analysis as you move into Phase 2. Explore each phase, broken down into manageable smaller steps in the [online Adaptation Planning Guide](https://resilientca.org/apg/intro/).
+ 
+> Cities and Counties in California are required to address climate adaptation and vulnerability in their safety elements. For more information on the requirements see the State's 2020 Adaptation Planning Guide, the General Plan Guidelines Chapter 4, or Government Code section 65302(g). (Senate Bill 379, Jackson, 2015).
+ 
+The way that you use climate data will depend on your particular needs, risk tolerance, and planning context. To support adaptation planning, the State of California has produced guidance materials, including:
+ 
+- 2020 Adaptation Planning Guide [<https://resilientca.org/apg/>]
```

Comment:
@elehmer is there a reason why we're not using the descriptive text with the link like one normally would with markdown? e.g. `[link text](link url)`?

Reply:
Cleaned this up with https://github.com/berkeley-gif/caladapt-website-2021/pull/35/commits/c90509fbfe1de41341d3fedf17249f128933a43c